kratos/tests/cpp_tests/geometries/test_quadrature_point_geometry.cpp

```diff
@@ -104,5 +133,86 @@ namespace Kratos {
             KRATOS_CHECK_EQUAL(p_this_quadrature_point->ShapeFunctionsValues().size2(), 3);
         }
 
+        KRATOS_TEST_CASE_IN_SUITE(QuadraturePointGeometry2dCopyConstructor, KratosCoreFastSuite)
+        {
+            auto p_this_quadrature_point = GenerateQuadraturePointGeometry2();
+
+            QuadraturePointGeometry<Node<3>, 2, 2> geom
+                = QuadraturePointGeometry<Node<3>, 2, 2>(*p_this_quadrature_point);
+
+            KRATOS_CHECK_EQUAL(geom.size(), 3);
+            KRATOS_CHECK_EQUAL(geom.WorkingSpaceDimension(), 2);
+            KRATOS_CHECK_EQUAL(geom.LocalSpaceDimension(), 2);
+            KRATOS_CHECK_EQUAL(geom.Dimension(), 2);
+
+            KRATOS_CHECK_EQUAL(geom.ShapeFunctionsValues().size1(), 1);
+            KRATOS_CHECK_EQUAL(geom.ShapeFunctionsValues().size2(), 3);
+
+            KRATOS_CHECK_MATRIX_NEAR(
+                geom.ShapeFunctionsValues(),
+                p_this_quadrature_point->ShapeFunctionsValues(),
+                1e-6);
+        }
+
+        KRATOS_TEST_CASE_IN_SUITE(QuadraturePointGeometry2dCopyConstructorDifferentPointType, KratosCoreFastSuite)
+        {
+            auto p_this_quadrature_point = GenerateQuadraturePointGeometry2();
+
+            QuadraturePointGeometry<Point, 2, 2> geom
```

Comment:
this calls first the copy and then the assignment CTor
I would change it to only call the Copy Ctor:
QuadraturePointGeometry<Point, 2, 2> geom(*p_this_quadrature_point)

Reply:
sure, changed it! Thanks!